facade/service.go

```diff
@@ -1073,9 +1084,6 @@ func (f *Facade) scheduleService(ctx datastore.Context, tenantID, serviceID stri
 		default:
 			svc.DesiredState = int(desiredState)
 		}
-		if err := f.fillServiceConfigs(ctx, svc); err != nil {
```

Comment:
Per our discussion, I'm going to replace the logic in this visitor function to schedule the service without calling `updateService()`. That way, I can leave service configs out of the equation entirely.